Lib/test/test_ioctl.py

```diff
@@ -66,8 +66,6 @@ def test_ioctl_mutate_2048(self):
         # Test with a larger buffer, just for the record.
         self._check_ioctl_mutate_len(2048)
 
-    # TODO: RUSTPYTHON
-    @unittest.expectedFailure
```

Comment:
fyi, I think this test is flaky, it passes locally for me too but I don't think it passes on CI

Reply:
surprise! it passed!!! I don't know why either